build.gradle

```
@@ -82,8 +82,20 @@ java.withSourcesJar()
 
 shadowJar {
     manifest.from(MANIFEST)
```

```suggestion
    manifest.from(MANIFEST)
    minimize()
```

Maybe worth doing this, since these deps are non-API in the shaded jar? dunno